src/NuGet.Core/NuGet.PackageManagement/ConsoleProjectContext.cs

```diff
@@ -6,9 +6,8 @@
 using System.Xml.Linq;
 using NuGet.Common;
 using NuGet.Packaging;
-using NuGet.ProjectManagement;
 
-namespace NuGet.CommandLine
+namespace NuGet.ProjectManagement
```

Comment:
I class from `nuget.exe` to `NuGet.PackageManagement` because it is already public, is that ok?